environment/src/main/java/com/sequenceiq/environment/environment/validation/EnvironmentValidatorService.java

```diff
@@ -246,4 +253,24 @@ public ValidationResult validateStorageLocation(String storageLocation, String s
         }
         return resultBuilder.build();
     }
+
+    public ValidationResult validateEncryptionKeyUrl(EnvironmentCreationDto creationDto, EntitlementService entitlementService) {
+        ValidationResultBuilder resultBuilder = ValidationResult.builder();
+        if (AZURE.name().equalsIgnoreCase(creationDto.getCloudPlatform())) {
+            String encryptionKeyUrl = creationDto.getParameters().getAzureParametersDto().getAzureResourceEncryptionParametersDto().getEncryptionKeyUrl();
```

Comment:
can getAzureParametersDto be null?

Reply:
Yes, I'll update this to handle the null exception. Thank you :)